src/common/passwordEncryption.ts

```diff
@@ -10,4 +11,18 @@ export class PasswordEncryption {
       hashedPassword,
     };
   }
+
+  async correctPassword(candidatePassword: string, userPassword: string) {
+    return await bcrypt.compare(candidatePassword, userPassword);
+  }
```

Comment:
usually we try to use a verb when naming a function, so this is a bit weird because it looks like you are correcting the password. Maybe you could do this: 

```suggestion
  async isPasswordCorrect(candidatePassword: string, userPassword: string) {
    return await bcrypt.compare(candidatePassword, userPassword);
  }
```

Reply:
done